vhosting/pagespeed.sls

```diff
@@ -0,0 +1,15 @@
+# Install pagespeed module
+{%- set webserver = salt['pillar.get']('vhosting:server:webserver', 'nginx') %}
+
+{%- if webserver == 'nginx' %}
+{%- set pkg_name = 'nginx-module-pagespeed' %}
```

Comment:
This might be irrelevant or not even possible, but what about indentation here?

Reply:
Jinja can be rather picky if you start indenting, that's why we don't for blocks like these.